phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/ViewEngines.java

```diff
@@ -90,7 +90,7 @@ public ViewEngines(ThemeManager themeManager, String enginesPath) {
 
         Bindings.bindContent(availableEngines.getTabs(), mappedSubCategoryTabs);
 
-        this.initializeSidebar();
+        this.setSidebar(this.sidebar);
```

Comment:
I'm not sure. `initializeSidebar` requires that you have knowledge about the internal structure of `MainWindowView` when deriving from it (i.e. that it has a field `sidebar`).